internal/faustjs.org/docs/blocks/gutenberg-blocks.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: gutenberg-blocks-provider
+title: Getting Started with Gutenberg Blocks Provider
+---
+
+# Getting Started with Gutenberg Blocks Provider
+
+## Quick Start
+
+:::note
+
+Make sure you have completed the initial setup for Faust at [Getting Started](/docs/getting-started).
+
+:::
+
+Install the blocks:
+
+```bash
+npm i @faustwp/blocks
+```
+
+Open ```_app.js``` and import the blocks provider:
+
+```js
+import { WordPressBlocksProvider } from '@faustwp/blocks';
+```
+
+Then, inside your templates you need to pass on the ```contentBlocks``` data in your ```WordPressBlocksViewer```:
+
+```js
+import { WordPressBlocksViewer } from '@faustwp/blocks';
+
+const { title, content, featuredImage, date, author, contentBlocks } =
+    props.data.post;
+const blocks = flatListToHierarchical(contentBlocks);
```

Comment:
`flatListToHierarchical` is not provided in the source code example. Maybe reference the code with a link.